ballista/core/src/execution_plans/distributed_query.rs

```diff
@@ -238,6 +238,7 @@ impl<T: 'static + AsLogicalPlan> ExecutionPlan for DistributedQueryExec<T> {
                 self.session_id.clone(),
                 query,
                 self.config.default_grpc_client_max_message_size(),
+                self.config.clone(),
```

Comment:
we already have `gpc_max_message_size` as a parameter can we pass other needed values as parameters rather than passing config